src/tile.rs

```diff
@@ -16,7 +16,7 @@ pub struct Tile {
     pub id: u32,
     pub image: Option<Image>,
     pub properties: Properties,
-    pub objectgroup: Option<ObjectGroup>,
+    pub objectgroup: Option<(ObjectLayer, Properties)>,
```

Comment:
Not sure about this: Tile may contain an object group, and this member doesn't include things such as name or ID (Those are stored in `Layer`). If we used `Layer` then the usage of `LayerType::ObjectLayer` would be implicit and it would create invalid states (e.g. tile has a tile layer in it).

Reply:
If I am correct, this object group refers to the tile collisions, so no data that is in the `Layer` struct actually matter here, not even the properties (at least in 99% in the cases) since it's inaccessible from the editor anyways.

Reply:
Hmm, indeed even the properties are not accessible in the editor as far as I know, so potentially just `Option<ObjectGroup>` would suffice.